app_dart/lib/src/service/config.dart

```diff
@@ -97,7 +97,7 @@ class Config {
   }
 
   // Dimension list defined in .ci.yaml.
-  List<String> get dimensionList => <String>['os', 'device_os', 'device_type', 'mac_model'];
+  static List<String> get dimensionList => <String>['os', 'device_os', 'device_type', 'mac_model'];
```

Comment:
Should this be moved into `Target`? I can't see a reason why these need to be accessible outside of `Target`.

Reply:
Done.